app/src/main/kotlin/com/rockspin/bargainbits/data/repository/deals/BBGameDealRepository.kt

```diff
@@ -0,0 +1,65 @@
+package com.rockspin.bargainbits.data.repository.deals
+
+import com.rockspin.bargainbits.data.BBDatabase
+import com.rockspin.bargainbits.data.models.DbDeals
+import com.rockspin.bargainbits.data.models.DealSortType
+import com.rockspin.bargainbits.data.models.GameDeal
+import com.rockspin.bargainbits.data.models.GroupedGameDeal
+import com.rockspin.bargainbits.data.rest_client.GameApiService
+import io.reactivex.Observable
+import io.reactivex.Single
+
+/**
+ * Created by valentin.hinov on 29/05/2017.
+ */
+class BBGameDealRepository(
```

Comment:
I agree in principle - I inherited the "BB" prefix from what we had in the code before. I'm open to changing it after but for now all repository-like implementation objects should have it for consistency sake.